docs/admin.rst

```diff
@@ -1,9 +1,71 @@
 Administrator Guide
 =====================
 
-At this point, you (the administrator) should have your own username and
-password, plus two-factor authentication through either the Google
-Authenticator app on your smartphone or a YubiKey.
+You (the administrator) should have your own username and password, plus
+two-factor authentication through either the Google Authenticator app on
+your smartphone or a YubiKey.
+
+.. _Responsibilities:
+
+Responsibilities
+----------------
+
+The SecureDrop architecture contains multiple hardened servers, and while we have
+automated many of the installation and maintenance tasks, a skilled Linux
+administrator and some manual intervention is required to responsibly run the system.
+
+This section outlines the tasks the administrator is responsible for in order to
+ensure that the SecureDrop server continues to be a safe place for sources to
+talk to journalists.
+
+Keep your SecureDrop Server Updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You should maintain awareness of SecureDrop updates and take any required
+manual action if requested in the `SecureDrop Release Blog`_. We recommend
+subscribing to the `SecureDrop RSS Feed`_ to stay apprised of new updates.
+
+Most often, the SecureDrop server will automatically update via apt. However,
+occasionally you will need to run the Ansible playbooks. We will inform you in
+the release blog when this is the case. If you are onboarded to our support portal,
+we will let you know in advance of major releases if manual intervention will be
+required.
+
+.. _`SecureDrop Release Blog`: https://securedrop.org/news
+.. _`SecureDrop RSS Feed`: https://securedrop.org/news/feed
+
+Keep your Network Firewall Updated
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I feel like this section could provide a little more detailed instructions. For older instances, it seems like admins need to manually update then upgrade to the latest 2.3.X release. For newer instances (2.3.0 and beyond), it seems like auto-updates are default, but upgrades must still be done manually. They should probably just consult the instructions below for more explicit instructions on doing so, as they should be better than what we can provide.

https://doc.pfsense.org/index.php/Firmware_Updates
https://doc.pfsense.org/index.php/Upgrade_Guide

...I wonder if there's a way to do auto-upgrades like `cron-apt`. I think most admins are not regularly upgrading their pfSense box. And while I think adding this documentation may increase the amount who do, I don't think it will resolve the problem in the same way automatic upgrades would.